.buildkite/steps/deploy.sh

```diff
@@ -3,13 +3,14 @@
 set -euo pipefail
 
 echo --- :hammer: Installing tools
-apk add helm yq skopeo git --quiet --no-progress
```

Comment:
We don't need these anymore as we are not building the container in this script.